src/Command/Ingester/LineItemIngesterCommand.php

```diff
@@ -184,7 +185,7 @@ private function createLineItem(array $rawLineItem): LineItem
             $rawLineItem['label'],
             $rawLineItem['uri'],
             $rawLineItem['slug'],
-            LineItem::STATUS_ENABLED,
+            $rawLineItem['status'] ?? LineItem::STATUS_ENABLED,
```

Comment:
When i try to ingest with status on header but not specifying a value for the column, it's giving error instead of enabling by default. Could you please check it?

```
[ERROR] Invalid line item status received: ''
```

Reply:
Or maybe it's just expected?

Reply:
Yes it is. __IF__ you have `status` column defined in csv, it must have a valid value (validated inside `LineItem` entity). If you don't specify it in the csv file, then default value `enabled` will be used.